core/gui/src/app/common/type/workflow.ts

```diff
@@ -24,7 +23,6 @@ export interface WorkflowContent
     operators: OperatorPredicate[];
     operatorPositions: { [key: string]: Point };
     links: OperatorLink[];
-    groups: PlainGroup[];
```

Comment:
please pay attention to comments/docs. for example line 14 in this file.

Reply:
updated.